File: CS-Examples/10_Conversion/ToMhtml.cs
using System;
using System.Windows.Forms;
using Spire.Doc;
using Spire.Doc.Documents;

namespace ToMhtml
{
    public partial class Form1 : Form
    {
        public Form1()
        {
            InitializeComponent();
        }

        private void button1_Click(object sender, EventArgs e)
        {
            //Create word document
            Document document = new Document();

            //Load the file from disk.
            document.LoadFromFile(@"..\..\..\..\..\..\Data\ToMhtml.docx");

            //Save to RTF file.
            document.SaveToFile("ToMhtml-out.mhtml", FileFormat.Mhtml);

            //Dispose the document
            document.Dispose();

            //Launching the MS Word file.
            WordDocViewer("ToMhtml-out.mhtml");
        }

        private void WordDocViewer(string fileName)
        {
            try
            {
                System.Diagnostics.Process.Start(fileName);
            }
            catch { }
        }

    }
}

File: VB-Examples/10_Conversion/ToMhtml.vb
Imports Spire.Doc
Imports Spire.Doc.Documents

Namespace ToMhtml
	Partial Public Class Form1
		Inherits Form
		Public Sub New()
			InitializeComponent()
		End Sub

		Private Sub button1_Click(ByVal sender As Object, ByVal e As EventArgs) Handles button1.Click
			'Create word document
			Dim document As New Document()

			'Load the file from disk.
			document.LoadFromFile("..\..\..\..\..\..\Data\ToMhtml.docx")

			'Save to RTF file.
			document.SaveToFile("ToMhtml-out.mhtml", FileFormat.Mhtml)

			'Dispose the document
			document.Dispose()

			'Launching the MS Word file.
			WordDocViewer("ToMhtml-out.mhtml")
		End Sub

		Private Sub WordDocViewer(ByVal fileName As String)
			Try
				Process.Start(fileName)
			Catch
			End Try
		End Sub

	End Class
End Namespace
